library-valcon-internship/package.json

```diff
@@ -20,6 +20,7 @@
     "react-router-dom": "^6.8.2",
     "react-scripts": "5.0.1",
     "react-select": "^5.7.0",
+    "react-toastify": "^9.1.1",
```

Comment:
**Optional:** You can investigate what does ^ and ~ mean when adding dependencies.